sumo-api/sumo/metrics/mesh_evaluator.py

```diff
@@ -80,8 +80,8 @@ def sample_mesh(faces, density=625):
     faces (np array - 3*N x D) -  matrix representing vertices and faces with
     [X, Y, Z, ...].  faces[0:3, :] is the first face. N is the
     number of faces.  D >=3 (columns beyond 3 are interpolated, too)
-    density (float) - Number of points per square meter to sample.
-    Default 625 gives one point every 4 cm on average.
+    density (float) - Number of points per unit square surface to sample.
```

Comment:
Note that all objects in SUMO are actual size, so there is no need to remove meter here.

Reply:
I was thinking not all datasets would be in meters, so unit square surface would encompass those cases.